app/build.gradle

```diff
@@ -21,30 +21,31 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.appcompat:appcompat:1.3.1'
 
-    implementation 'com.google.firebase:firebase-core:11.0.4'
-    implementation 'com.google.firebase:firebase-auth:11.0.4'
-    implementation 'com.google.firebase:firebase-database:11.0.4'
-    implementation 'com.google.firebase:firebase-storage:11.0.4'
-    implementation 'com.google.android.gms:play-services-maps:11.0.4'
+    implementation 'com.google.firebase:firebase-core:19.0.1'
+    implementation 'com.google.firebase:firebase-auth:21.0.1'
+    implementation 'com.google.firebase:firebase-database:20.0.2'
+    implementation 'com.google.firebase:firebase-storage:20.0.0'
+    implementation 'com.google.android.gms:play-services-maps:17.0.1'
     implementation 'com.firebaseui:firebase-ui-database:0.4.0'
```

Comment:
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.firebaseui:firebase-ui-database:0.4.0.
 
<img src="https://cdn.discordapp.com/attachments/893056008786694144/894285476394057738/unknown.png" style="height: 100px; width:100px;"/>

Please build the application first to test whether up-gradation working properly or not.